### ISSUE TITLE
pop-up chat box requires scrolling

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#959]
Fix pop-up chat to not require so much scrolling.

jrx:
> when i open the chat box i have to scroll a lot, you mentioned you fixed this one?

I may need to ask jrx for more info, but I should futz around first.

**Comment by ctm** [comment]
I still see this myself sometimes while playing in the evening tournament, but I haven't reproduced it locally. OTOH, I haven't tried hard, either.

In yesterday's game, I chose to pop-out the window after switching the filter from comments-only to all-messages and I had to scroll down. I didn't think at the time that the scroll-down arrow appeared either when chat was docked or after it was popped-out, but I didn't think to check.

I have a bunch of business work to do today, but if I need a break from that, I may wind up doing some code inspection to see if I can see the bug, because if I can, then reproducing it and fixing it should indeed be trivial. Beyond that, I'll pay closer attention to what's going on in the evening game when I see it.

**Comment by ctm** [comment]
I spent a little time with the demo earlier and couldn't reproduce it. Notably even if the docked chat wasn't at the bottom, opening the pop-up brought us to the bottom. So, what I saw yesterday couldn't have merely been a matter of me not being at the bottom and not noticing it.

One thing I didn't do just now (or, AFAIK when I've tried to reproduce this previously) is play as many hands in the demo as I likely did in yesterday's tournament. There may be some weird interaction between the amount of text in the chat box and the scrolling. Another possibility might be based on the length of the longest line. Perhaps if someone types a long line that wraps it causes trouble. I mention that because I don't think I've tried that locally when reproducing. Both of these are grasping at straws, but _something's_ causing the weird behavior.
